### DISCUSSION TITLE
How to get `Simulation`?

### DISCUSSION THREAD

**Discussion by blair-ahlquist** [stride3d/stride#1742] [Q&A] [answered]
This page shows how to do a raycast: https://github.com/stride3d/stride-docs/blob/master/en/manual/physics/raycasting.md/. How does one get a handle on a `Simulation` instance?

**Comment by IXLLEGACYIXL** [comment, marked as answer]
By 
```
ScriptComponent thing = blabla;
thing.GetSimulation();
```
Here is an example
https://github.com/NexStandard/NexCoreExtension/blob/main/CameraRayExtension.cs
